common/rhal_base.h

```diff
@@ -10,11 +10,19 @@ extern "C" {
 #include <stdint.h>
```

Comment:
remove "stdint" header file from here, as it is already included in common_utilities.h

Reply:
Fixed